### DISCUSSION TITLE
Aliucord Bot Client v1.0.0

### DISCUSSION THREAD

**Discussion by CE1CECL** [ChrisEric1/ChrisEric1.github.io#32] [Announcements]
See https://github.com/ChrisEric1/ChrisEric1.GitHub.io/issues/29#issuecomment-2041167347 for more info

<hr /><em>This discussion was created from the release <a href='https://github.com/ChrisEric1/ChrisEric1.GitHub.io/releases/tag/1.0.0a'>Aliucord Bot Client v1.0.0</a>.</em>

**Comment by Ushie** [comment]
Hey, how was this client made? Is there a fork of Aliucord that achieves this?

**Reply by CE1CECL** [reply]
I explained how its built here: https://github.com/ChrisEric1/ChrisEric1.GitHub.io/issues/29#issuecomment-3677815706
